Chat/Chat.cpp

```diff
@@ -96,7 +96,7 @@ void Chat::PutMsg(const UserMsg& msg)
 void Chat::PrintSomeoneList(std::vector<std::string>& list)
 {
     chat_mutex_.Lock();
-    for (int i = 0; i < list.size(); i++)
+    for (int i = 0; i < (int)list.size(); i++)
```

Comment:
yeah, but you can create "size_t i" variable. Can you check if there will be some warnings in that case?
I mean "for(size_t i = 0; i <...."